### ISSUE TITLE
[BITV]: Add more visibility on background buttons

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#40682]
This is an accesibility issue.

For now we have different preview-buttons with odifferent backgrounds. Selected one have to be more visible with enough contrasts. A black check-icon is not visible on different backgrounds and that have to be replaced with another solution.

![Screenshot from 2023-09-28 16-46-02](https://github.com/nextcloud/server/assets/6078378/dffb7d07-3e9d-4953-91fd-29034747a9db)

@nimishavijay would we have a quick design and implementation change in this place? Which way is a "cheapest" one? Thank you!

**Comment by JuliaKirschenheuter** [comment]
Thank you @nimishavijay!
